### DISCUSSION TITLE
Image download,  network unreachable

### DISCUSSION THREAD

**Discussion by soloam** [antonym/netboot.xyz#1413] [General]
**Describe the bug**
When I bot on proxmox to the network boot I go to the netboot xyz menu and when I click to start a boot, for example popos or debian, I get a message "netboot.xyz network unreachable" (screenshot sbelow)

**To Reproduce**
1. Probably it will not reproduce on your system, but any image that I select I get this error

**Expected behavior**
Install the image that I selected

**Screenshots**
![image](https://github.com/netbootxyz/netboot.xyz/assets/11949987/6410953b-63ec-41c7-94d5-cf21c36d06dc)

<img width="503" alt="image" src="https://github.com/netbootxyz/netboot.xyz/assets/11949987/babff63d-eecf-4a8c-b6fd-be607c1efcc0">

**Additional context**
- I have a docker install. 
- If I go to the webui of the netboot.xyz I'm able to download the image
- If I do a nslookup on the iPXE console I allways get IPv6 IP's... But I don't have IPv6, don't know if related

**Comment by ravsit-activum** [comment]
I had the same thing. I turned off ipv6 address resolution in my dns server (adguard home) and now it works.

**Comment by soloam** [comment]
I'm running pfsense! I will try to find where that setting is

**Reply by pmagid** [reply]
Did you ever find what is needed in PfSense to address this...

**Comment by GaMeRaM** [comment]
> I had the same thing. I turned off ipv6 address resolution in my dns server (adguard home) and now it works.

If anyone is having problems on OpenWRT the fix is to go to 
`Network -> DHCP and DNS -> Advanced Settings`
and check 
`Filter IPv6 AAAA records`

**Reply by nyanyaon** [reply]
Thanks it work for me

**Comment by mdsina** [comment]
Same here on Mikrotik.
Trying to filter record but no luck.
Process hangs on downloading network installer.
After that got the same error of network timeout 
![изображение](https://github.com/netbootxyz/netboot.xyz/assets/1622569/b7cb333a-dfa0-49ce-a1b4-531858137815)

**Comment by GaMeRaM** [comment]
> Same here on Mikrotik. Trying to filter record but no luck. Process hangs on downloading network installer. After that got the same error of network timeout ![изображение](https://private-user-images.githubusercontent.com/1622569/298279985-b7cb333a-dfa0-49ce-a1b4-531858137815.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU3NTMwMjMsIm5iZiI6MTcwNTc1MjcyMywicGF0aCI6Ii8xNjIyNTY5LzI5ODI3OTk4NS1iN2NiMzMzYS1kZmEwLTQ5Y2UtYTFiNC01MzE4NTgxMzc4MTUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDEyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAxMjBUMTIxMjAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODI4MzU1ZTE5MzU2Yjc4M2QwZWE3OTBkODYyODQwYjUyZjFiNTA4N2Q4MGI3NGZkNWZhMmYwN2I1Zjg4ZTRhYyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.TNTLke-0WwD_W6eeQSUlOvubkgvtsznQSZqSJQ91PE8)

Seems like it's a different problem. Original problem happens instantly while yours seems like partly working. Can you check your network connection consistency? Maybe try to boot smaller image?

Also you can check if you can download the file from your main computer

**Comment by mdsina** [comment]
> Can you check your network connection consistency? Maybe try to boot smaller image?

I've tried different things. Nothing works.

Initially netboot looks fine
![изображение](https://github.com/netbootxyz/netboot.xyz/assets/1622569/116d28a4-af95-438d-8ae8-240ee9151225)
![изображение](https://github.com/netbootxyz/netboot.xyz/assets/1622569/f0e96def-5d81-4b4b-990f-0efbcab43698)

But when I choose any kind of OS, it hangs on downloading.
![изображение](https://github.com/netbootxyz/netboot.xyz/assets/1622569/6f5303ee-f2c0-440a-a222-f762263e86b7)

I can download it manually just fine without any errors, but not from netboot
Any suggestions?

**Comment by dennco** [comment]
Hi @ all,
i have the same error with my Mikrotik like the  topic creator but ipv6 is deactivated in my "mikrotik-router" but manually i can reach from this VM the sites and download them .. 
Any tips ?

**Reply by pmagid** [reply]
I gave up and downloaded all the images as local assets and that "worked" for me... Would strongly prefer not to have to do this.

**Reply by futurealecks** [reply]
I downloaded a few local assets and it still hangs downloading locally. It gets 10-30 percent in and hangs.

**Comment by judders** [comment]
Anyone find a fix for this. New to netbootxyz. downloading the image stalls at a different % each time. using openwrt, and vmware for testing.

**Reply by NiKiZe** [reply]
Make sure you have a working IPv6 network, if not, make sure that you dont have anything that claims that you can reach IPv6 internet. Some devices is configured to announce IPv6 reachability even if it isn't available. But honestly, every ISP and every network today should just have IPv6, better to fix the bugs (on the router level) and make it work than to disable them.

**Comment by ser** [comment]
Why the trick with removal of IPv6 from DNS is not in FAQ?? It took me 2 hours to resolve this s..t
